File: Euclid/Messages/Outgoing/ITEMS.cs
using Euclid.Game;
using Euclid.Network.Streams.Util;
using System.Collections.Generic;

namespace Euclid.Messages.Outgoing
{
    public class ITEMS : IMessageComposer
    {
        private List<ActiveItem> items;

        public ITEMS(List<ActiveItem> items)
        {
            this.items = items;
        }

        public override void Write()
        {
            foreach (var item in items)
            {
                item.Serialise(Data);
            }
        }
    }
}

File: euclid/src/main/java/euclid/messages/outgoing/ITEMS.java
package euclid.messages.outgoing;

import euclid.game.item.ActiveItem;
import euclid.messages.IMessageComposer;

import java.util.List;

public class ITEMS extends IMessageComposer {

    private final List<ActiveItem> items;

    public ITEMS(List<ActiveItem> items) {
        this.items = items;
    }

    @Override
    public void write() {
        for (ActiveItem item : items) {
            item.serialise(getData());
        }
    }
}
